src/modules/Builder/pages/BuilderApplet/BuilderApplet.utils.tsx

```diff
@@ -22,6 +22,7 @@ import {
 } from 'shared/state';
 import { getDictionaryText, getEntityKey, Path } from 'shared/utils';
 import { ItemResponseType } from 'shared/consts';
+import { PerformanceTasks } from 'modules/Builder/features/Activities/Activities.types';
```

Comment:
mb export types in Activities folder?

Reply:
there is default export there
![image](https://github.com/ChildMindInstitute/mindlogger-admin/assets/116339462/e479174b-c330-4ed6-9315-1ef9b9128b20)